js/utils/DOMUtils.js

```diff
@@ -4,18 +4,18 @@ export const createDiv = (innerText = '') => {
   const element = document.createElement('div');
   element.innerHTML = innerText;
   return element;
-}
+};
```

Comment:
Exporting (and assignment of a function as const) are both statements, so a semicolon is polite

Reply:
We would not want to offend.